doc/pages/Manual.md

```diff
@@ -652,6 +651,54 @@ other system).
 
 `FOO += ""`, `FOO := ""`, etc. are all ignored - i.e. opam never adds empty segments to an existing variable.
 
+#### Environment update portability
+<a id="env-update-rewrite"></a>
+
+Some fields define an environment update portability specification. In the opam
+file it is the [`x-env-path-rewrite:`](#opamfield-x-env-path-rewrite) in the
+form:
+
+```BNF
+<environement-rewrites> ::= { <environement-rewrite> }*
+<environement-rewrite>  ::= <ident> <bool>
+                          | <ident> <separator-formula> <format-formula>
+
+<separator-formula>     ::= <separator-formula> <logop> <separator-formula>
+                          | ( <separator-formula> )
+                          | <separator> { { <distrib-formula> }* }
+<format-formula>        ::= <format-formula> <logop> <format-formula>
+                          | ( <format-formula> )
+                          | <format> { { <distrib-formula> }* }
```

Comment:
As far as i could see, only `separator-formula = <separator>` and `format-formula = <format>` can be shown in the tests. I think it would be nice to have the other syntaxes tested for completeness.

Reply:
added